services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala

```diff
@@ -30,8 +29,8 @@ class WorkbenchHealthMonitorServiceActor(val serviceConfig: Config, globalConfig
 
   override lazy val subsystems: Set[MonitoredSubsystem] = Set(DockerHub, EngineDb, Papi, Gcs)
 
-  private lazy val Gcs = MonitoredSubsystem("GCS", checkGcs _)
```

Comment:
iirc this fuckery was due to one of the linters we have (eta expansion?) which makes me wonder what changed to make it not necessary?

Reply:
- The original `MonitoredSubsystem("GCS", checkGcs _)` works in the linter, but gives a yellow box in intellij instead of the green checkmark.
- The intellij "fix", `MonitoredSubsystem("GCS", checkGcs)`, trips up the linter with the "eta expansion" message you referred to.
- The replacement `MonitoredSubsystem("GCS", () => checkGcs())` gives a green intellij checkmark and passes the scalac linter!

Reply:
Sweet. As an aside, the new signature (which I swear I tried, but perhaps something else was different when I did) is exactly why I wanted to make them `IO` and not `Future` but that opened a can of worms I didn't have time to dig into